serde_with/tests/serde_as/lib.rs

```diff
@@ -136,6 +137,41 @@ fn test_option() {
     );
 }
 
+#[test]
+fn test_bound() {
+    #[serde_as]
+    #[derive(Debug, Serialize, Deserialize, PartialEq)]
+    struct S(#[serde_as(as = "_")] Bound<u32>);
```

Comment:
This does not test your code, but instead the already existing Serialize and Deserialize implementations for Bound. You want `as = "Bound<_>"`.

Can you please add a test that converts the inner type somehow? For example `Bound<DisplayFromStr>`.

Reply:
> This does not test your code, but instead the already existing Serialize and Deserialize implementations for Bound. You want `as = "Bound<_>"`.

Oh, that makes sense! Done.

By the way, wouldn't it be the same with [`test_option`](https://github.com/jonasbb/serde_with/pull/655/files#diff-db1bb1fa4de5910d07d9a9d12ae3276fda201487e23f3006933052fac23613adR119) then?

> Can you please add a test that converts the inner type somehow? For example `Bound<DisplayFromStr>`.

Done.

Reply:
Yes, you are right. That test looks wrong too.